config/lobby/db/002_update_tables

```diff
@@ -0,0 +1,6 @@
+alter table ta_users add column bcrypt_password character(60) check (char_length(bcrypt_password)=60);
```

Comment:
Given the `002` file in #2355, this file will probably be `003_something`.  These scripts are applied sequentially, so you'll need to decide which PR you want to merge first so you can build one script off the changes in the other.  Obviously, the changes may be totally orthogonal and can be worked independently, but testing should be done exactly as the database changes will be applied in production.

Also, the comment I made in #2355 about running this code within a transaction applies here, as well.